src/styles/buy.css

```diff
@@ -0,0 +1,25 @@
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin-top: 16px;
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+
+  border: 1px solid #00acdc;
+  background: #00acdc;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__buy:hover {
+  border: 1px solid #00acdc;
+  background-color: #fff;
+  color: #00acdc;
+}
```

Comment:
These styles belong to the card block. Maybe they should be in the card.css file instead.